validator/core/models.py

```diff
@@ -220,3 +220,44 @@ class NetworkStats(BaseModel):
     number_of_jobs_success: int
     next_training_end: datetime | None
     job_can_be_made: bool = True
+
+
+class ColumnPickerResponse(BaseModel):
+    field_instruction: str
+    field_output: str
+    field_input: str | None = None
+
+    def validate_columns_ok(self, sparse_columns: list[str], non_sparse_columns: list[str]) -> bool:
```

Comment:
Feels like a function outside of the model definition

Reply:
I can move it out, but I think it belongs in the class since it validates the class fields directly. Keeping it here makes it easy to see which columns are being validated and allows for intuitive usage with response.validate_columns_ok(). That said, if you feel strongly about moving it out, I’m happy to adjust.